subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -1722,13 +1747,70 @@ static inline int rx_demux_rx(memq_link_t *link, struct node_rx_hdr *rx)
 	return 0;
 }
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_BROADCASTER)
+static void term_evt_op_stop_cb(uint32_t status, void *param)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0, 0, &link, NULL, NULL};
+	struct ll_adv_set *adv;
+	struct ull_hdr *hdr;
+	uint32_t ret;
+
+	/* Ignore if race between thread and ULL */
+	if (status != TICKER_STATUS_SUCCESS) {
+		/* TODO: detect race */
+
+		return;
+	}
+	adv = param;
+	hdr = &adv->ull;
+	mfy.param = &adv->lll;
+
+	if (hdr->ref) {
```

Comment:
will this always be true by design? if so, is this check required?

Reply:
Removed this function. I copied this functionality from ull_adc.c (ticker_op_stop_cb). I assume we don't need to call lll_disable, as we do in ull_adv.c when ref count is 0. lll_disable probably gets called as a result of an HCI call?